lib/websocket/index.js

```diff
@@ -3,33 +3,28 @@
 // Handles everything to do with the websocket transport and message responders
 'use strict';
 
-var EventEmitter2, emitter;
-
-EventEmitter2 = require('eventemitter2').EventEmitter2;
-
-emitter = new EventEmitter2({
-  wildcard: true
-});
+var EventEmitter2 = require('eventemitter2').EventEmitter2,
+    emitter = new EventEmitter2({
+      wildcard: true
+    });
 
 module.exports = function(ss) {
-  var transport;
  
   // Return API
-  transport = require('./transport')(ss, emitter);
+  var transport = require('./transport')(ss, emitter);
  
   return {
     transport: transport,
     unload: function() {},
     load: function(httpServer, responders, eventTransport, sessionOptions) {
-      var id, responder, thisTransport;
-      thisTransport = transport.load(httpServer, sessionOptions);
+      var thisTransport = transport.load(httpServer, sessionOptions);
 
       // Dispatch incoming events to websocket clients      
       require('./event_dispatcher')(eventTransport, thisTransport, emitter);
 
       // Listen to incoming requests and invoke server.request      
-      for (id in responders) {
-        responder = responders[id];
+      for (var id in responders) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.